js/src/ui/Form/TypedInput/typedInput.js

```diff
@@ -61,8 +67,11 @@ export default class TypedInput extends Component {
   };
 
   componentWillMount () {
-    if (this.props.isEth && this.props.value) {
-      this.setState({ isEth: true, ethValue: fromWei(this.props.value) });
+    const { isEth, value } = this.props;
+
+    if (typeof isEth === 'boolean' && value) {
```

Comment:
I guess there is a specific reason for this, but isn't boolean validation done by `propTypes` already?